build/packer/ansible/roles/podman-setup/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+- name: pull the PMM image
+  command: podman pull {{ pmm_server_image_name }}
+
+- name: get srv directory path
+  shell: podman image inspect {{ pmm_server_image_name }} | jq -r .[0].GraphDriver.Data.UpperDir
+  register: image_path
+
+- name:
+  copy:
+    src: "{{ image_path.stdout }}/srv/"
+    dest: /srv/
+    mode: preserve
+    remote_src: yes
+
+- name: Copy content of 'srv' directory from image
+  synchronize:
+    src: "{{ image_path.stdout }}/srv/"
+    dest:  /srv/
+    recursive: yes
+  delegate_to: "{{ inventory_hostname }}"
```

Comment:
I'm not sure this hack is even necessary. However, if otherwise, let's leave a good comment explaining why.